src/styles/block/cards.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  width: 100%;
+  display: grid;
+  padding-top: 50px;
+  margin: 0 auto;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  place-content: center;
+
+  &__section {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: $background-color;
+    transition: transform 0.3s ease;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
```

Comment:
The `card__section` class seems to be the main container for the card's content. Consider renaming it to something more descriptive, like `card__content` or `card__body`. This will make your code more readable and understandable.